### PR TITLE
Copy snapshots to test bundle to solve SPM warning

### PR DESCRIPTION
**Pull request by valeriyvan**
There's a warning about 16 unhandled resources when running build in command line. PR addresses this warning by adding to package manifest instruction to copy resources to test bundle and exclude Info.plist.

```
warning: 'wescan': found 16 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/VisionRectangleDetectorTests/testCorrectlyDetectsAndReturnsQuadilateral.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/ReviewViewControllerTests/testImageIsCorrectlyRotated90.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/ReviewViewControllerTests/testEnhancedImage.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/ReviewViewControllerTests/testImageIsCorrectlyRotated180.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/UIImageTests/testRotateRightFacingImageCorrectly.1.png
/Users/***/WeScan/Tests/WeScanTests/Info.plist
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/UIImageTests/testRotateLeftFacingImageCorrectly.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/VisionRectangleDetectorTests/testCorrectlyDetectsAndReturnsQuadilateralPixelBuffer.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/ReviewViewControllerTests/testDemoImageIsCorrect.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/CIRectangleDetectorTests/testCorrectlyDetectsAndReturnsQuadilateral.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/UIImageTests/testRotateUpFacingImageCorrectly.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/UIImageTests/testRotateDefaultFacingImageCorrectly.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/ReviewViewControllerTests/testImageIsCorrectlyRotated360.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/ReviewViewControllerTests/testImageIsCorrectlyRotated270.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/UIImageTests/testRotateDownFacingImageCorrectly.1.png
/Users/***/WeScan/Tests/WeScanTests/__Snapshots__/UIImageTests/testRotateImageCorrectly.1.png
```

### CODE REVIEW

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **CIRectangleDetectorTests.testCorrectlyDetectsAndReturnsQuadilateral():**<br/>failed - Snapshot does not match reference.

@−
"/Users/vagrant/git/Tests/WeScanTests/__Snapshots__/CIRectangleDetectorTests/testCorrectlyDetectsAndReturnsQuadilateral.1.png"
@+
"/Users/vagrant/Library/Developer/CoreSimulator/Devices/D7CA41A8-547E-4DC0-9C53-FF5A41928CA5/data/tmp/CIRectangleDetectorTests/testCorrectlyDetectsAndReturnsQuadilateral.1.png"

To configure output for a custom diff tool, like Kaleidoscope:

    SnapshotTesting.diffTool = "ksdiff"

Actual perceptual precision 0.69906247 is less than required 0.97
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **VisionRectangleDetectorTests.testCorrectlyDetectsAndReturnsQuadilateral():**<br/>failed - Snapshot does not match reference.

@−
"/Users/vagrant/git/Tests/WeScanTests/__Snapshots__/VisionRectangleDetectorTests/testCorrectlyDetectsAndReturnsQuadilateral.1.png"
@+
"/Users/vagrant/Library/Developer/CoreSimulator/Devices/D7CA41A8-547E-4DC0-9C53-FF5A41928CA5/data/tmp/VisionRectangleDetectorTests/testCorrectlyDetectsAndReturnsQuadilateral.1.png"

To configure output for a custom diff tool, like Kaleidoscope:

    SnapshotTesting.diffTool = "ksdiff"

Newly-taken snapshot does not match reference.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>'gray' was deprecated in iOS 13.0: renamed to 'UIActivityIndicatorView.Style.medium'</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  'init(source:)' was deprecated in iOS 12.0: Core Image Kernel Language API deprecated. (Define CI_SILENCE_GL_DEPRECATION to silence these warnings)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>'blackTranslucent' was deprecated in iOS 13.0: Use UIBarStyleBlack and set the translucent property to YES instead.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>'isAutoStillImageStabilizationEnabled' was deprecated in iOS 13.0</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>'jpegPhotoDataRepresentation(forJPEGSampleBuffer:previewPhotoSampleBuffer:)' was deprecated in iOS 11.0</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>'blackTranslucent' was deprecated in iOS 13.0: Use UIBarStyleBlack and set the translucent property to YES instead.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>WeScanTests: Executed 60 tests (2 failed, 0 retried, 0 skipped) in 14.252 seconds</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Slowest test: RectangleFeaturesFunnelTests/testAddAlternateImage() (4.870s)</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Slowest test: RectangleFeaturesFunnelTests/testAddPreviouslyDisplayedRect() (3.141s)</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Slowest test: RectangleFeaturesFunnelTests/testAddMinUnderThreshold() (3.022s)</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/9fcba28a-acec-4545-8f86-a9836c2bfcfd" target="_blank">Bitrise</a>
  </td>
    </tr>
  </tbody>
</table>

## Code Coverage Report
| Name | Coverage ||
| --- | --- | --- |
WeScan | 37.73% | ⚠️


### SwiftLint found issues

| Severity | File | Reason |
| -------- | ---- | ------ |
Warning | Package.swift:28 | Colons should be next to the identifier when specifying a type and next to the key in dictionary literals (`colon`) |
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 419b4d469148bda0947732ce845119a3cbe7f8c1
</p>

**Review by AvdLee — APPROVED** [review]
Thanks for your contribution!

**Review by BasThomas — APPROVED** [review]

File: Package.swift [hunk 1]
```diff
@@ -25,8 +25,10 @@ let package = Package(
                 "WeScan",
                 .product(name: "SnapshotTesting", package: "swift-snapshot-testing")
             ],
+            exclude:["Info.plist"],
```
Comment (BasThomas) [inline Package.swift:28]:
```suggestion
            exclude: ["Info.plist"],
```

But oh well 🤷

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 3.0.0-beta.1](https://github.com/WeTransfer/WeScan/releases/tag/3.0.0-beta.1) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

**Pull request merged**